generate.rb

```diff
@@ -94,6 +94,7 @@
   --template eisvogel \
   --table-of-contents \
   --toc-depth 6 \
+  --listings -H wrap \
```

Comment:
Using listing has a bunch of downsizes and doesn't really fix the issue as there is some internal limitations to pandoc anyway.

https://github.com/Wandmalfarbe/pandoc-latex-template/issues/122#issuecomment-538011690